### PR TITLE
Update gradle

### PR DESCRIPTION
**Pull request by acmlira**
Often it is always a good idea to update Gradle. In this case, we want to give more stability to the Android platform and considering these issues: #92 and #196, we had to act. This is the first change that aims to make the build for android more secure and pleasant for the developer. Changes:

- Update Gradle to enable Android plugin update;
- Update the Android plugin for Gradle (so that the NDK used is as new as possible);
- No more errors related to the NDK version (at least without log).

### CODE REVIEW

File: TotalCrossVM/android/app/build.gradle [hunk 1]
```diff
@@ -46,6 +45,10 @@ android {
             }
         }
     }
+
+    packagingOptions {
+        pickFirst "**/libskia.so"
```
Comment (flsobral) [inline TotalCrossVM/android/app/build.gradle:50]: has this been tested with both armv7 and arm64? can you confirm it really picks the correct version?

File: TotalCrossVM/android/app/build.gradle [hunk 2]
```diff
@@ -14,8 +14,7 @@ apply plugin: 'com.android.application'
 def buildType // stores build type, debug or release
 
 android {
-    compileSdkVersion 28
-    ndkVersion "21.0.6113669"
```
Comment (flsobral) [inline TotalCrossVM/android/app/build.gradle:18]: we should enforce the usage of a ndk version to make sure the results are the same, either keep 21.0.6113669 or move to the latest one

**Review by flsobral — CHANGES_REQUESTED** [review]

File: TotalCrossVM/android/app/build.gradle (hunk 1 above)
Comment (acmlira) [reply TotalCrossVM/android/app/build.gradle:50]: Was tested only in arm64. On the solution: it works but can be modified with further tests in the future. The best solution suggested in the post that I linked at the commit does not work in TotalCross (tested).

File: TotalCrossVM/android/app/build.gradle (hunk 2 above)
Comment (acmlira) [reply TotalCrossVM/android/app/build.gradle:18]: The gradle plugin takes care of that

**Review by flsobral — APPROVED** [review]

**Pull request merged**